extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/restclient/ClientTracingFilter.java

```diff
@@ -98,10 +98,5 @@ public void filter(ClientRequestContext requestContext, ClientResponseContext re
         }
     }
 
-    private static final TextMapSetter<MultivaluedMap<String, Object>> SETTER = new TextMapSetter<MultivaluedMap<String, Object>>() {
-        @Override
-        public void set(MultivaluedMap<String, Object> carrier, String key, String value) {
-            carrier.add(key, value);
-        }
-    };
+    private static final TextMapSetter<MultivaluedMap<String, Object>> SETTER = MultivaluedMap::add;
```

Comment:
Converted the anonymous class to a lambda expression

Reply:
Any particular reason?

For me, it's a little harder to immediately understand what's happening

Reply:
K, reverted no issues